modules/@apostrophecms/admin-bar/ui/apos/components/TheAposAdminBarLocale.vue

```diff
@@ -41,11 +86,30 @@ export default {
         modifiers: [ 'icon-right', 'no-motion' ],
         type: 'quiet'
       };
+    },
+    filteredLocales(input) {
+      return this.locales.filter(locale => {
+        return (
+          locale.label.toLowerCase().indexOf(this.search.toLowerCase()) > -1
```

Comment:
Oh cool, search was going to be pushed back til later, but it never hurts to get ahead.

ES2015 note: could use `.includes` in place of `indexOf` and the `-1` check,  but that's just a nitpick.

Reply:
More importantly though, please check the locale name as well as the locale label, in case they don't put the locale's official name in the label but the user searches for it.